src/main/java/org/broadinstitute/hellbender/utils/test/GenomicsDBTestUtils.java

```diff
@@ -40,4 +43,19 @@ public synchronized static void runOnGenomicsDBArray(GenomicsDBTestAction test,
 
         void run() throws IOException;
     }
+
+    public static <T> void assertCondition(Iterable<T> actual, Iterable<T> expected, BiConsumer<T,T> assertion){
```

Comment:
This seems like a very generic method rather than a GenomicsDB-specific one -- can you move to `BaseTest`?